vendor/github.com/openshift/installer/data/data/bootstrap/files/etc/mdsd.d/mdsd.env.template

```diff
@@ -0,0 +1,15 @@
+MONITORING_GCS_ENVIRONMENT={{ .LoggingConfig.Environment }}
+MONITORING_GCS_ACCOUNT=AROClusterLogs
```

Comment:
arguably this and MONITORING_GCS_NAMESPACE should be passed in, but it's probably OK...

Reply:
added field, hardcoded in create.go since the env interface doesn't have a configurable Monitoring Namespace.